includes/compat/class-wc-stripe-subscriptions-helper.php

```diff
@@ -54,6 +71,11 @@ public static function get_some_detached_subscriptions() {
 				continue;
 			}
 
+			// Filter subscriptions not renewing in the next month
+			if ( $subscription->get_time( 'next_payment' ) > ( time() + MONTH_IN_SECONDS ) ) {
```

Comment:
Not blocking: Just to be extra safe, should we check for today + 1 month + 1 day? That way we catch subscriptions that renew in one month but later in the day.
```suggestion
			if ( $subscription->get_time( 'next_payment' ) > ( time() + MONTH_IN_SECONDS + DAY_IN_SECONDS ) ) {
```